libs/ibl/CMakeLists.txt

```diff
@@ -43,8 +43,6 @@ target_link_libraries(${TARGET} math utils)
 # ==================================================================================================
 if (MSVC)
     target_compile_options(${TARGET} PRIVATE /fp:fast)
-else()
-    target_compile_options(${TARGET} PRIVATE -ffast-math $<$<PLATFORM_ID:Linux>:-fPIC>)
```

Comment:
You lost `-ffast-math`, and we really need it

Reply:
Fixed.  I should probably move fast math to the global level too.